### DISCUSSION TITLE
How to update to latest stable version

### DISCUSSION THREAD

**Discussion by Roeliooo** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#842] [Q&A]
Hi,

How to update to the last stable version instead of the beta channel ?

If I run this command on my raspberry, it updates itself to beta 3.2 (this version doesn't work for me)
```
cd domoticz/plugins/zigbee2mqtt
git pull

```

Thanks!

**Comment by acaonweb** [comment]
There something about downgrade, in the main page
Plugin downgrade

Go to plugin folder and pull older version
cd domoticz/plugins/zigbee2mqtt
git reset --hard f8c22fc330229b39f06ff11bbfcea38e3a492306
Restart domoticz or disable and re-enable plugin in 'Setup -> Hardware' (clearing browser cache might be necessary!)

It seems this downgrade to stable 3.0.0
